FirebaseCoreInternal.podspec

```diff
@@ -0,0 +1,74 @@
+Pod::Spec.new do |s|
+  s.name             = 'FirebaseCoreInternal'
+  s.version          = '8.13.0'
```

Comment:
```suggestion
  s.version          = '8.15.0'
```

Reply:
These are kind of wrong either way and will go to 9.0.0 shortly